doc/pr2_tutorials/planning/scripts/doc/move_group_python_interface_tutorial.rst

```diff
@@ -16,7 +16,7 @@ The entire code can be seen :codedir:`here in the moveit_pr2 github project<plan
 The launch file
 ^^^^^^^^^^^^^^^
 The entire launch file is `here
-<https://github.com/ros-planning/moveit_tutorials/tree/master/doc/pr2_tutorials/planning/launch/move_group_python_interface_tutorial.launch>`_
+<https://github.com/ros-planning/moveit_tutorials/tree/indigo-devel/doc/pr2_tutorials/planning/launch/move_group_python_interface_tutorial.launch>`_
```

Comment:
reference ext_links in conf.py

Reply:
eg :codedir:`here<move_group_python_interface/launch/move_group_python_interface_tutorial.launch>`